src/alire/alire.ads

```diff
@@ -51,8 +51,8 @@ package Alire with Preelaborate is
    function "+" (S : UString) return String
    renames UStrings.To_String;
 
-   subtype Project_Character is Character
-      with Static_Predicate => Project_Character in
+   subtype Crate_Character is Character
+      with Static_Predicate => Crate_Character in
          'a' .. 'z' | '0' .. '9' | '_' | Extension_Separator;
```

Comment:
Completely unrelated: Why hyphen is not allowed in crate name?

Reply:
When the index was Ada-based I tried to keep things more closer to Ada syntax (indeed the hyphens in the toml syntax ruffle me the wrong way, but since it's a matter of taste...) So no actual reason now I'd say.

Reply:
I'm mostly curious, I do like hyphens in project names better than underscores (`lvgl-ada` vs `lvgl_ada`) but since allowing both would be a source of confusion let's keep it how it is.